calendar/calendar-util/util/src/java/org/sakaiproject/util/CalendarUtil.java

```diff
@@ -72,6 +78,7 @@
 	Date dateOctober = null;
 	Date dateNovember = null;
 	Date dateDecember = null;
+	private Map<String, String> eventIconMap = new HashMap<String, String>();
```

Comment:
I'm happy for this to be merged without this and my other comments being addressed.  They were nice to haves really.